builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +228,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
+		response := &logical.Response{
+			Data: map[string]interface{}{
```

Comment:
@vishalnayak I haven't used the other ssh endpoints, cause up until the CA they weren't useful to me. You're right that HTTPRawBody is probably the better way to return this, easier for consistency, scripts, etc. Ultimately, whichever return format is chosen is fine with me.